docs/content/docs/integrations/nessie.md

```diff
@@ -35,15 +35,15 @@ See [Project Nessie](https://projectnessie.org) for more information on Nessie.
 
 The `iceberg-nessie` module is bundled with Spark and Flink runtimes for all versions from `0.11.0`. To get started
 with Nessie and Iceberg simply add the Iceberg runtime to your process. Eg: `spark-sql --packages
-org.apache.iceberg:iceberg-spark3-runtime:{{% icebergVersion %}}`. 
+org.apache.iceberg:iceberg-spark-runtime-3.2_2.12:{{% icebergVersion %}}`. 
```

Comment:
as discussed on Slack this isn't currently supported by the Nessie Spark SQL Extensions as those are still running with Spark 3.1 and below. Once this is fixed on the Nessie side, I'll update the docs here